src/tanuki/language_models/language_model_manager.py

```diff
@@ -129,20 +136,27 @@ def get_generation_case(self, args, kwargs, function_description, llm_parameters
         # no examples needed, using a finetuned model. Dont save to finetune dataset
         if is_distilled_model and suitable_for_distillation:
             prompt = self.construct_prompt(f, args, kwargs, [], distilled_model)
+            if func_hash not in self.current_generators:
+                self.current_generators[func_hash] = {"model": "finetuned_model_placeholder", "examples": []}
             return prompt, distilled_model, suitable_for_distillation, True
 
         else:
             aligns = self.function_modeler.get_symbolic_alignments(function_description.__hash__(), max=16)
             examples = [f"Inputs:\nArgs: {align['args']}\nKwargs: {align['kwargs']}\nOutput: {align['output']}" for align in
                  aligns]
             
+            if func_hash not in self.current_generators:
+                self.current_generators[func_hash] = {"model": "", "examples": examples}
```

Comment:
Can we change the variable name `current_generators` to something more intuitive?